cmd/bclient/fileutil/fileutil.go

```diff
@@ -0,0 +1,154 @@
+package fileutil
+
+// A catchall package for bclient file operations and data structures.
+// May need decomposition if too unwieldy
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"sync"
+
+	"github.com/antonholmquist/jason"
+)
+
+var (
+	UpLoadDone     sync.WaitGroup
+	UseRemoteList  bool = true
+	localFileList  *FileList
+	remoteFileList *FileList
+	FilesWalked    = make(chan string)
+	rootPrefix     string
+	verbose        bool
+)
```

Comment:
Moved UpLoadDone     sync.WaitGroup into the main, and deleted the
fileutil functions that used it.

I'll leave verbose util we find a good  logging program


On Fri, Jan 22, 2016 at 2:29 PM, Don Brower <notifications@github.com>
wrote:

> In cmd/bclient/fileutil/fileutil.go
> <https://github.com/ndlib/bendo/pull/56#discussion_r50579616>:
>
> > +	"path"
> > +	"path/filepath"
> > +	"sync"
> > +
> > +	"github.com/antonholmquist/jason"
> > +)
> > +
> > +var (
> > +	UpLoadDone     sync.WaitGroup
> > +	UseRemoteList  bool = true
> > +	localFileList  *FileList
> > +	remoteFileList *FileList
> > +	FilesWalked    = make(chan string)
> > +	rootPrefix     string
> > +	verbose        bool
> > +)
>
> too many globals!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ndlib/bendo/pull/56/files#r50579616>.
>